src/exists.js

```diff
@@ -217,7 +217,15 @@ function testModulePath(value, fileDir) {
   return `Cannot find module '${value}'`;
 }
 
+// Gets extensions array from rule config options, e.g. [1, { "extensions" : ['.jsx'] }]
```

Comment:
add the rule to the e.g.: `"require-path-exists/exists": [1, { "extensions": ['.jsx'] }]`